### PR TITLE
Add set rcv buffer for syncd rpc

### PR DESCRIPTION
**Pull request by pavel-shirshov**

### CODE REVIEW

File: dockers/docker-syncd-mlnx-rpc/Dockerfile [hunk 1]
```diff
@@ -44,5 +44,6 @@ COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 COPY ptf_nn_agent.conf /etc/supervisor/conf.d/ptf_nn_agent.conf
 
 ENTRYPOINT service rsyslog start \
+    && sysctl -w net.core.rmem_max=509430500 \
```
Comment (qiluo-msft) [inline dockers/docker-syncd-mlnx-rpc/Dockerfile]: magic number

File: dockers/docker-syncd-mlnx-rpc/ptf_nn_agent.conf [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [program:ptf_nn_agent]
-command=/usr/bin/python /opt/ptf_nn_agent.py --device-socket 1@tcp://0.0.0.0:10900   -i 1-3@Ethernet12
+command=/usr/bin/python /opt/ptf_nn_agent.py --device-socket 1@tcp://0.0.0.0:10900 -i 1-3@Ethernet12 --set-iface-rcv-buffer=109430400
```
Comment (qiluo-msft) [inline dockers/docker-syncd-mlnx-rpc/ptf_nn_agent.conf]: magic number

File: dockers/docker-syncd-mlnx-rpc/Dockerfile (hunk 1 above)
Comment (qiluo-msft) [inline dockers/docker-syncd-mlnx-rpc/Dockerfile]: why do 'sysctl' inside container? should we do it on host as init script?

Comment (pavel-shirshov) [inline dockers/docker-syncd-mlnx-rpc/Dockerfile]: What do you want to see here?

Comment (pavel-shirshov) [inline dockers/docker-syncd-mlnx-rpc/Dockerfile]: Because it makes sense and bound to the processes which lives inside of the container.

Comment (qiluo-msft) [inline dockers/docker-syncd-mlnx-rpc/Dockerfile]: why you choose this magic number?

Comment (qiluo-msft) [inline dockers/docker-syncd-mlnx-rpc/Dockerfile]: If bounded, should roll back the effect after container dies. How about container's systemd file if really bounded.

Comment (pavel-shirshov) [inline dockers/docker-syncd-mlnx-rpc/Dockerfile]: I found this number experimentally. I can round it to some value, let say 500000000. Will it work for you?

Comment (pavel-shirshov) [inline dockers/docker-syncd-mlnx-rpc/Dockerfile]: Does it matter for the container which is used only in tests?

Comment (qiluo-msft) [inline dockers/docker-syncd-mlnx-rpc/Dockerfile]: yes, thanks!

Comment (qiluo-msft) [inline dockers/docker-syncd-mlnx-rpc/Dockerfile]: no big deal for test.

File: dockers/docker-syncd-rpc/Dockerfile [hunk 3]
```diff
@@ -44,5 +44,6 @@ COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 COPY ptf_nn_agent.conf /etc/supervisor/conf.d/ptf_nn_agent.conf
 
 ENTRYPOINT service rsyslog start \
+    && sysctl -w net.core.rmem_max=509430500 \
```
Comment (lguohan) [inline dockers/docker-syncd-rpc/Dockerfile]: for this line, what about add it to the sonic-mgmt or put it into the systemctl file?

Comment (pavel-shirshov) [inline dockers/docker-syncd-rpc/Dockerfile]: What's wrong with this line? Can you please explain it further?

I don't see any rationale to put this line into the host-machine level (systemd). In this case we will have two base systems (for tests and for production).

Also I don't see any rationale to change this systemctl before the tests, then restart ptf_nn_agent, then change it back again and restart ptf_nn_agent again.

**Pull request merged**